spec/models/movement_spec.rb

```diff
@@ -3,9 +3,16 @@
 require 'rails_helper'
 
 RSpec.describe Movement, type: :model do
-  subject { FactoryBot.build(:movement) }
+  subject { FactoryBot.build(:movement, direction: 'arrival') }
```

Comment:
Is the subject still used? Should that line be deleted

Reply:
@Br3nda It's used on line 11